ivadomed/utils.py

```diff
@@ -403,6 +404,17 @@ def format_path_data(path_data):
     return path_data
 
 
+def similarity_score(a: str, b: str) -> float:
```

Comment:
I would recommend we be explicit in naming this function as `string_similarity_score` instead. This is because "similarity score" has different meanings in ML (eg cosine similarity, distance-based similarity like Euclidean, L1, etc.), just to avoid unnecessary confusion.